js/toolbar/toolbar.view.js

```diff
@@ -75,19 +93,22 @@ function ToolbarView(controller, config){
     }
 
     // Toggle menu content 
-    this.toggleDisplayMenuContent = function(type, list){
+    this.toggleDisplayMenuContent = function(type, object){
         
         this._navMenuContentElem.className = (this._navMenuContentElem.className.indexOf("expand") >= 0) ? "" : "expand";
 
         // if it's expand, insert the content
         if(this._navMenuContentElem.className.indexOf("expand") >= 0){
             switch(type){
                 case "channelList":
-                    this.renderChannelContent(list);
+                    this.renderChannelContent(object);
                     break;
                 case "annotationList":
-                    this.renderAnnotationGroupContent(list);
+                    this.renderAnnotationGroupContent(object);
                     break;
+                // case "annotationTool":
```

Comment:
we should remove this one. we shouldn't leave commented code paths like this without any comments why it's commented. If the feature is not needed, we just remove the code.